group-owner.html

```diff
@@ -0,0 +1,299 @@
+
+<li>adjust and save rendering settings under their own name</li>
+
+<li>draw and save ROIs on the images</li>
+
+<li>use the data in analysis and figures.</li>
+
+</ul>
+
+<p>In addition to the other interactions, group owners can:</p>
+
+<ul class="bullet-list">
+
+<li>change the permissions level of the group (except for promoting to Read-Write, which is an admin privilege)</li>
+
+<li>import data for other users in the group</li>
+
+<li>delete or move the data belonging to any members of the group</li>
+
+<li>using the Command Line Interface, change the owner of the data, for example if the original owner has left the lab.</li>
+
+</ul>
+
+<p>Details of the Command Line Interface <a href="http://www.openmicroscopy.org/site/support/omero/users/cli/index.html" target="_blank" class="external-url">OMERO.cli</a> are in the OME documentation.</p>
+
+<p>Details on accessing OMERO with the OMERO.web client are in the Using the <a href="http://help.openmicroscopy.org/web-client.html" target="_blank" class="external-url">OMERO.web Client</a> section of User Help.</p>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">If permissions are reduced to Private when one member has created output from another member's image in the group using a script, e.g. made a projection, the permissions change will fail.</p>
+</div><!-- End note -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>Default groups</h3>
+
+<p>A user who is a member of more than one group can set any group as their default group. This group will be then shown in the data tree each time they log in. For details on how users can change their default group see Changing your default group in the <a href="http://help.openmicroscopy.org/sharing-data.html#default" target="_blank" class="external-url">Sharing Data</a> section of User Help.</p>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="add"></a>Adding a user to your group</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+  <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<ol class="step">
+
+<li><p class="first">Click on the menu with your name on the right in the top toolbar.</p>
+
+<p>Select <span class="bold">User Settings</span> in the menu.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/groupOwnerEditGroup.jpg" class="align-center" src="images/groupOwnerEditGroup.jpg" style="width: 100%;" />
```

Comment:
The screenshots in this section are the wrong way round - this one illustrates the 'My groups' tab mentioned under the next point.